x-pack/plugins/apm/public/components/app/service_overview/service_overview_instances_chart_and_table.tsx

```diff
@@ -24,25 +30,78 @@ interface ServiceOverviewInstancesChartAndTableProps {
   serviceName: string;
 }
 
+const INITIAL_STATE = {
+  items: [] as Array<{
```

Comment:
Can we use something from `ApiReturnType` here?

Reply:
Not really, since the type is changed from `number | null | undefined` to `number` here https://github.com/cauemarcondes/kibana/blob/apm-time-comparison-instances/x-pack/plugins/apm/public/components/app/service_overview/service_overview_instances_chart_and_table.tsx#L123-L127